core/include/crypto/crypto_impl.h

```diff
@@ -138,6 +138,7 @@ TEE_Result crypto_aes_cmac_alloc_ctx(struct crypto_mac_ctx **ctx);
 TEE_Result crypto_des3_cmac_alloc_ctx(struct crypto_mac_ctx **ctx);
 #else
 CRYPTO_ALLOC_CTX_NOT_IMPLEMENTED(aes_cmac, mac)
+CRYPTO_ALLOC_CTX_NOT_IMPLEMENTED(des3_cmac, mac)
```

Comment:
Should be part of the same commit that adds the `crypto_des3_cmac_alloc_ctx()` prototype, right?

Reply:
Yes, that is correct.
I was about to reorder commits anyway as my initial approach was a bit naive (missed the mbedTLS part)